### ISSUE TITLE
msvc2015 c++11

### ISSUE THREAD

**Issue by fabiencastan** [alicevision/openMVG#287]
msvc2015 has C++11 <chrono> header, leading to compile failure. the problem is in CXX11.cmake on the line [here](https://github.com/alicevision/openMVG/blob/popart_develop/src/cmakeFindModules/CXX11.cmake#L9): `if((MSVC AND (MSVC10 OR MSVC11 OR MSVC12))`

It doesn't list any newer compilers. it should use MSVC_VERSION instead.

**Comment by zvrba** [comment]
Here's how to check for `>=` http://stackoverflow.com/questions/16667017/cmake-express-the-greater-or-equal-statement. This is the relevant table of versions from cmake docs:

```
1200 = VS  6.0
1300 = VS  7.0
1310 = VS  7.1
1400 = VS  8.0
1500 = VS  9.0
1600 = VS 10.0
1700 = VS 11.0
1800 = VS 12.0
1900 = VS 14.0
```

**Comment by pmoulon** [comment]
I made a fix there some weeks ago https://github.com/openMVG/openMVG/blob/develop_matching/src/cmakeFindModules/CXX11.cmake#L9
But it would certainly better to use a >= if close in order to support seamlessly the future VS version.

**Comment by simogasp** [comment]
closed with alicevision/openMVG#289
